index.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <!-- TODO: Wihtin the html page I need to get rid of commented code or get that code to work. Most of the commented code wants deals with the category and title filters. I had this working previously but since adding new things they are not working. I either need to get them working or remove them all together. They are making my code hard to read.  -->
```

Comment:
As you commented yourself, getting rid of corpse code will help you see what is going on in your code better and see what isn't working/necessary. I had a lot of that prior to class_09 branch, and I still have some, but some were parts of old code from prior to refactoring that wasn't going to be used for or adapted to anything. getting rid of these helped me really clean up code.